cpg_workflows/stages/large_cohort.py

```diff
@@ -75,11 +75,12 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
                 vds_paths.append(tmp_query_res)
                 sg_ids_in_vds = sg_ids_in_vds + tmp_sg_ids_in_vds
 
-        # Get SG IDs from the cohort object itself, rather than call Metamist.
-        # Get VDS IDs first and filter out from this list
-        cohort_sgs: list[SequencingGroup] = cohort.get_sequencing_groups(only_active=True)
+        if not config_retrieve(['combiner', 'merge_only_vds'], None):
+            # Get SG IDs from the cohort object itself, rather than call Metamist.
+            # Get VDS IDs first and filter out from this list
+            cohort_sgs: list[SequencingGroup] = cohort.get_sequencing_groups(only_active=True)
 
-        new_sg_gvcfs: list[str] = [str(sg.gvcf) for sg in cohort_sgs if sg.id not in sg_ids_in_vds]
+            new_sg_gvcfs: list[str] = [str(sg.gvcf) for sg in cohort_sgs if sg.id not in sg_ids_in_vds]
```

Comment:
you'll need to create an empty `new_sg_gvcfs` list variable if you pass the `merge_only_vds` test, otherwise the next logical test will try and check a var that wasn't instantiated yet

Reply:
Fair point, oops

Reply:
Done